### PR TITLE
Add CLV  dct:identifier

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Fix: #153

### CODE REVIEW

**Review by giorgialodi — COMMENTED** [review]
It should be also included in the RDF/XMl version and in the JSON-LD versione

File: Ontologie/CLV/latest/CLV-DBGT-AP_IT.jsonld [hunk 1]
```diff
@@ -13,47 +12,17 @@
   },
   "@graph": [
     {
-      "@id": "clvapit:Address",
+      "@id": "clvapit:InternalAccess",
```
Comment (giorgialodi) [inline Ontologie/CLV/latest/CLV-DBGT-AP_IT.jsonld:15]: @ioggstream what's this change?

**Review by giorgialodi — COMMENTED** [review]
@ioggstream why changing the alignment to DGBT?

**Comment by giorgialodi** [comment]
@ioggstream did you add dct:identifier in the CLV-AP_IT.rdf and CLV-AP_IT.jsonld?

**Comment by ioggstream** [comment]
> @ioggstream why changing the alignment to DGBT? 

I copied the latest/ version to 1.0/. Should they be the same?

**Comment by ioggstream** [comment]
> @ioggstream did you add dct:identifier in the CLV-AP_IT.rdf and CLV-AP_IT.jsonld?

Yes, can you see it in lode?

File: Ontologie/CLV/latest/CLV-DBGT-AP_IT.jsonld (hunk 1 above)
Comment (ioggstream) [reply Ontologie/CLV/latest/CLV-DBGT-AP_IT.jsonld:15]: Good spot. This should not be changed!

**Comment by ioggstream** [comment]
@giorgialodi thanks for the review. Now it should be fixed!

**Review by giorgialodi — APPROVED** [review]
Ok it seems to me that you recovered the previous code. That's fine then. We can merge

**Pull request merged**